src/Counter.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+
+function Counter({fruitName, image}) {
+    const [amount, setAmount] = useState(0);
+
+    function plusClickHandler() {
+        setAmount(amount + 1)
+        return amount
+    }
+
+    function minusClickHandler() {
+        if (amount >= 1) {
+            setAmount(amount - 1)
+            return amount
+        }
+    }
+    // ik krijg deze functie niet zo dat ik alle counters aangeroepen krijg,
+    // en  om alle fruitsoorten nou met de hand uit te gaan schrijven lijkt me ook niet helemaal "DRY"
+    //alle tips zijn welkom!
```

Comment:
Het is lastig om vanuit een hoger niveau state aan te passen. Wat we dan vaak zeggen is dat je het dan moet 'liften'. Je zult dan de useState met functies een niveau hoger moeten zetten. De minusClickHandler en plusClickHandler kun je dan weer meegeven aan dit component